mmv1/products/dialogflow/Generator.yaml

```diff
@@ -0,0 +1,236 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'Generator'
+description: |
+  LLM generator.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/agent-assist/docs'
+  api: 'https://cloud.google.com/dialogflow/es/docs/reference/rest/v2beta1/projects.generators'
```

Comment:
This product is actually using the v3 API https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/dialogflowcx/product.yaml